ui-support-persistence/src/main/java/org/dcsa/uisupport/persistence/repository/FacilityRepository.java

```diff
@@ -0,0 +1,21 @@
+package org.dcsa.uisupport.persistence.repository;
+
+import org.dcsa.skernel.domain.persistence.entity.Facility;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface FacilityRepository extends JpaRepository<Facility, UUID> {
+  @Query("""
+    select facility from Facility facility
+    where (:unLocationCode is null or facility.unLocationCode = :unLocationCode)
+    and facility.smdgCode is not null
+    """)
+  List<Facility> findFacilities(String unLocationCode);
```

Comment:
This method does not appear to be used? If it is not used, lets not introduce it in the first place.

Reply:
no it is removed